### PR TITLE
Switch to GHA

### PR DESCRIPTION
**Pull request by ihostage**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -124,16 +120,19 @@ lazy val javaClusteredChat = (project in file("samples/java/clustered-chat"))
...
-    "headerCheckAll",
-    "scalafmtSbtCheck",
-    "scalafmtCheckAll",
+  (
+    List(
+      "headerCheckAll",
+      "scalafmtSbtCheck",
+      "scalafmtCheckAll",
+    ) ++
+      List(scalaChat, scalaMultiRoomChat, scalaClusteredChat, javaChat, javaMultiRoomChat, javaClusteredChat)
```
Comment (mkurz) [inline build.sbt:135]: 👍

File: .travis.yml [hunk 2]
```diff
@@ -1,57 +0,0 @@
...
-
-services:
-  - xvfb
-
-addons:
-  chrome: stable
-  apt:
-    packages:
-      - unzip
-      - dbus-x11
```
Comment (mkurz) [inline .travis.yml:11]: Very nice the GHA VM's provide these packages.

File: build.sbt [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
 import Dependencies.Scala213
 import play.core.PlayVersion.{ current => playVersion }
 
-val AkkaVersion = "2.5.28"
-
 lazy val runChromeWebDriver = taskKey[Unit]("Run the chromewebdriver tests")
 
 val macwire = "com.softwaremill.macwire" %% "macros" % "2.3.4"
-val lombok  = "org.projectlombok"        % "lombok"  % "1.18.8"
+val lombok  = "org.projectlombok"        % "lombok"  % "1.18.8" % Provided
```
Comment (mkurz) [inline build.sbt:9]: Good catch 👍

File: .travis.yml (hunk 2 above)
Comment (ihostage) [reply .travis.yml:11]: I'm not sure 😄 Just I migrated tests to [webdrivermanager](https://github.com/bonigarcia/webdrivermanager) and used a headless mode 
https://github.com/playframework/play-socket.io/blob/60a5dec8eade3c28c6b753510e11157667ce8197/src/test/scala/play/socketio/RunSocketIOTests.scala#L36

Comment (ihostage) [reply .travis.yml:11]: That why now we don't need install Chrome driver manually and use X11 for tests 😄

Comment (mkurz) [reply .travis.yml:11]: > That why now we don't need install Chrome driver manually and use X11 for tests 😄

https://github.com/actions/virtual-environments/blob/ubuntu20/20220508.1/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers 😉

**Pull request merged**